config/profiler/requirements.txt

```diff
@@ -1,12 +1,14 @@
 pip==20.3.3
-wheel==0.35.1
-pyYaml==5.4
-pytest-html==3.0.0
-sagemaker==2.23.0
-pre-commit==2.6.0
-awscli==1.18.203
-pytest==6.1.2
-pytest-cov==2.10.1
+awscli==1.19.102
+boto3==1.17.102
+botocore==1.20.102
```

Comment:
The tests shouldn't manage dependencies for `boto3` and `botocore` since the tests don't consume them directly. `sagemaker` and `awscli` will manage them as dependencies, so let's remove these two packages.